visualizer.py

```diff
@@ -0,0 +1,323 @@
+#!/usr/bin/python3
+"""
+Author: Vertigo Designs, Ryan Summers
+
+Description: Bokeh application for serving Stabilizer stream visuals.
+"""
+import argparse
+import asyncio
+import enum
+import logging
+import socket
+from typing import List, Mapping
+
+import tornado
+
+import bokeh.plotting
+import bokeh.layouts
+import bokeh.document
+import bokeh.palettes
+import bokeh.io
+import bokeh.models
+import bokeh.server.server
+import numpy as np
+
+from miniconf import Miniconf
+from stabilizer.stream import StabilizerStream, Trace
+
+# The sample rate of stabilizer.
+SAMPLE_RATE_HZ = 100e6 / 128
+
+
+def _get_ip(broker):
```

Comment:
This is like a precursor to the entire TURN story for NAT traversal...
Can we (a) use asyncio's [`get_extra_info()`](https://docs.python.org/3/library/asyncio-protocol.html#asyncio.BaseTransport.get_extra_info) move it either into miniconf-py (preferrably) or into `stabilizer.tools`?

Reply:
We don't have the transport until after we've resolved the IP, so we can't use asyncio's `get_extra_info()` yet. I'm torn where this belongs - it doesn't feel like this is particularly related to `miniconf`, as it's more an implementation detail of `stabilizer`'s livestream functionality.